translations/monero-core_de.ts

```diff
@@ -140,19 +140,19 @@
     <message>
         <location filename="../components/DaemonConsole.qml" line="108"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>Schließen</translation>
     </message>
     <message>
         <location filename="../components/DaemonConsole.qml" line="119"/>
         <source>command + enter (e.g help)</source>
-        <translation type="unfinished"></translation>
+        <translation>Befehl + Enter (z.B. help)</translation>
     </message>
 </context>
 <context>
     <name>DaemonManagerDialog</name>
     <message>
         <source>Daemon doesn&apos;t appear to be running</source>
-        <translation type="vanished">Daemon scheint nicht aktiv zu sein</translation>
+        <translation type="vanished">Dämon scheint nicht aktiv zu sein</translation>
```

Comment:
I agree that the Node renaming should be handled seperately. I am not a big fan of the rather literal translation "Dämon", so I would prefer "Daemon". Do we have any opposing strong opinion? Otherwise, if nobody but me really cares we use Daemon, do we? ;)